.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="GoogleStyle" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="dremio" />
```

Comment:
nit: Can it be a separate PR? As the changes are independent. 

Also, I remember spotless uses GoogleStyle. So, it makes sense for Idea also to use GoogleStyle?

Reply:
oops - my mistake - removed this change from PR. Thanks for spotting! :+1: